src/tmx/TmxUtils/src/SNMPClient.h

```diff
@@ -46,16 +46,24 @@ namespace tmx::utils
         // struct that holds information about who we're going to be talking to
         // We need to declare 2 of these, one to fill info with and second which is
         // a pointer returned by the library
-        snmp_session session;
-        snmp_session *ss;
+        struct snmp_session session;
+        struct snmp_session *ss;
 
         /*Structure to hold all of the information that we're going to send to the remote host*/
-        snmp_pdu *pdu;
+        struct snmp_pdu *pdu;
+        /*Structure to hold response from the remote host*/
+        struct snmp_pdu *response;
 
         /*OID is going to hold the location of the information which we want to receive. It will need a size as well*/
         oid OID[MAX_OID_LEN];
         size_t OID_len = MAX_OID_LEN;
 
+        // Declare missing OID len definitions
+        #define USM_PRIV_PROTO_AES192_LEN 9
```

Comment:
Could we put this to the top of the file?

Reply:
That is definitely a good idea

Reply:
@jwillmartin Where are these missing from? Why did we need to define these?

Reply:
@paulbourelly999 These lengths weren't declared in the snmp library where other lengths were, so I added them in.

Reply:
Upon further investigation I think this needs to be limited to the class private scope. I do not thing we need access to these outside of the SNMP class scope.

Reply:
This is the suggestion from sonar code smell: https://sonarcloud.io/project/issues?issueStatuses=OPEN%2CCONFIRMED&pullRequest=652&id=usdot-fhwa-ops_V2X-Hub&open=AZOTp-UF_EH9KHLovh3Q
Do you mean that the macro is in private class scope if defined inside class, and would be in public scope if defined at the top of a header file?

Reply:
This is what I read from online:```
Scope Issues:
Macros don't respect C++ scoping rules. A macro defined inside a class is still visible outside of it, which can lead to unexpected behavior and naming collisions.
```